src/main/java/com/mateacademy/lessons12/rest/MateGroupServiceImpl.java

```diff
@@ -43,4 +43,16 @@ public Response addStudents(@PathParam("groupId") int groupId, Person person) {
 		return Response.status(Status.NOT_FOUND).build();
 	}
 
+	@Override
+	@PUT
+	@Path("/{groupId}/teacher")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response putTeacher(@PathParam("groupId") int groupId, Teacher teacher) {
```

Comment:
this method doesnt return mateGroup with new teacher, it just returns mateGroup

Reply:
Yes, you dont use teacher anything